Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -45,9 +46,29 @@ const configOptionTitle = (title, description) => {
   );
 };
 
+const timeFrames = [
+  { _id: 1, name: "seconds" },
+  { _id: 2, name: "minutes" },
+  { _id: 3, name: "hours" },
+  { _id: 4, name: "days" },
+];
+
+const sxButtons = {
+  width: 110,
+  height: 34,
+};
+
 const CreateNewMaintenanceWindow = () => {
   const [values, setValues] = useState({
     repeat: 1,
+    date: dayjs(),
+    startTime: dayjs(),
+    duration: {
+      magnitude: "",
+      unit: 1,
+    },
```

Comment:
Is there a need to have duration be an object? The simple solution here would be to separate duration value and unit into their own individual params. 

Like so:
`const [values, setValues] = useState({
    repeat: 1,
    date: dayjs(),
    startTime: dayjs(),
    duration: "60",
    unit: "minutes",
    friendlyName: "",
    AddMonitors: "",
  });`
  
  And then you can go ahead and update the input `value` and `onChange`:
  `<Field
        id="duration-value"
        placeholder="60"
        onChange={(e) => handleChange(e, "duration")}
        value={values.duration}
    />
    <Select
        onChange={(e) => handleChange(e, "unit")}
        id="duration-unit"
        items={timeFrames}
        value={values.unit}
    />`

Reply:
Done ✅